packages/dd-trace/test/platform/node/index.spec.js

```diff
@@ -381,8 +381,8 @@ describe('Platform', () => {
 
         platform = {
           _config: {
-            hostname: 'localhost',
```

Comment:
Why was this removed? I'm also not sure why removing it didn't fail any test.

Reply:
I've updated `platform/node/metrics.js` to use `config.dogstatsd.hostname` instead of `config.hostname`. 

https://github.com/DataDog/dd-trace-js/pull/995/files#diff-ded06582bc11034346decd1b6b2bcbbbR52

Tests are passing because the same value that was being passed in as `config.hostname` in tests is now being passed in as `config.dogstatsd.hostname`, which is set to the value of `config.hostname` when no value is explicitly set by the user.